deploy/update-and-deploy.sh

```diff
@@ -4,6 +4,7 @@ DH_PASS=${DH_PASS:?Docker repo password is not defined}
 DH_REPO_URL=${DH_REPO_URL:?Docker repo URL is not defined}
 GIT_REPO_URL=${GIT_REPO_URL:?Docker repo URL is not defined}
 GIT_BRANCH=${GIT_BRANCH:?Git branch is not defined}
+#VERSION - optional
```

Comment:
?

Reply:
all variable except VERSION is mandatory.
script use VERSION variable, but it can be omitted and set as empty string to preserve old behavior